build.sbt

```diff
@@ -290,7 +286,7 @@ lazy val common = Project(id = "common", base = file("modules/common"))
       }))
   )
   .settings(
-    wispSettings
+    unmanagedSourceDirectories in Compile += (sourceDirectory in Compile).value / ("scala-" + scalaBinaryVersion.value)
```

Comment:
this shouldn't be here? it's already set above in https://github.com/andypetrella/spark-notebook/pull/757/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR281